libs/core/src/lib/nested-list/nested-list/nested-list.directive.ts

```diff
@@ -172,16 +172,18 @@ export class NestedListDirective implements AfterContentInit, NestedListInterfac
     private _setAriaAttributes(level: number, lang: FdLanguage): void {
         this.nestedItems.forEach((item, i) => {
             item._ariaLevel = level;
-            item.linkItem.ariaDescribedby = this._nestedListHeader?.id || null;
-            item.linkItem._ariaLabel = this._translationResolver.resolve(lang, 'coreNestedList.linkItemAriaLabel', {
-                itemDetails: item.linkItem.getTitle(),
-                index: i + 1,
-                total: this.nestedItems.length,
-                selectedDescription:
-                    !this._nestedListStateService.selectable && item.linkItem.selected
-                        ? ', ' + this.ariaLabelSelected
-                        : ''
-            });
+            if (item.linkItem) {
```

Comment:
you can invert the condition so that if there's no link item, just return. It will be more readable I think.